package.json

```diff
@@ -28,7 +28,7 @@
     "prettier": "2.3.1"
   },
   "engines": {
-    "node": "^12.22.0 || >=14.15.0",
+    "node": ">=14.15.0",
```

Comment:
If we're only running our tests on Node 14, we shouldn't imply Node 12 support. 😄